CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set (CMAKE_USE_RELATIVE_PATHS 1)
 
 OPTION (ENABLE_DOCS "Build the documentation" OFF)
 OPTION (ENABLE_P4RUNTIME_TO_PD "Build the P4Runtime PD Generator" ON)
+OPTION (ENABLE_BMV2 "Build the BMV2 backend (required for the full test suite)" ON)
+OPTION (ENABLE_EBPF "Build the EBPF backend (required for the full test suite)" ON)
+OPTION (ENABLE_P4TEST "Build the P4Test backend (required for the full test suite)" ON)
+OPTION (ENABLE_P4C_GRAPHS "Build the p4c-graphs backend" ON)
```

Comment:
shouldn't you remove these if you've enabled them below now?

Reply:
I don't understand... the foreach loop is for extensions. These are built-in backends. Am I missing something?

Reply:
You're right. I read is as all backends being captured in that loop. Sorry about that.

Reply:
No worries. Thanks for reviewing so closely. =)

Reply:
Though I guess you /could/ have a loop over the backends directory to create all of these.  I guess the help messages would then be less informational (unless there was a way of pulling the message from a file in the directory in question...)